src/libs/actions/Report.js

```diff
@@ -1466,6 +1466,52 @@ function renameReport(reportID, reportName) {
         .finally(() => Onyx.set(ONYXKEYS.IS_LOADING_RENAME_POLICY_ROOM, false));
 }
 
+/**
+ * Update the policy room name.
+ *
+ * @param {Object} policyRoomReport
+ * @param {*} reportName The updated reportName
+ */
+function updatePolicyRoomName(policyRoomReport, reportName) {
+    const reportID = policyRoomReport.reportID;
+    const previousName = policyRoomReport.reportName;
+    const optimisticData = [
+        {
+            onyxMethod: CONST.ONYX.METHOD.MERGE,
+            key: `${ONYXKEYS.COLLECTION.REPORT}${reportID}`,
+            value: {
+                reportName,
+                pendingFields: {
+                    reportName: 'update',
```

Comment:
Let's create a CONST for `update` if there is not one already.

Reply:
Oh yeah, I added constants for that last week. Updated.